assets/sass/modules/_okta-footer.scss

```diff
@@ -39,6 +39,23 @@
       }
     }
 
+    a {
```

Comment:
is this for the widget footer? Where did these styles live before?

Reply:
This is for the <a> links in footer. This is needed here as the footer is outside the #okta-sign-in namespace.

Earlier, it was in _base.scss and it was not namespaced, which is why it worked even in footer as it applied globally to all <a>

Reply:
got it. This is the widget's footer right? Curious why is it outside of the widget namespace?